### PR TITLE
RIDI.Ridibooks version 0.9.4

### PR DESCRIPTION
**Pull request by alexkoala**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19941)

### CODE REVIEW

File: manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: RIDI.Ridibooks
+PackageVersion: 0.9.4
+PackageLocale: ko-KR
+Publisher: RIDI
```
Comment (ItzLevvie) [inline manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml:7]:
```suggestion
Publisher: RIDI Labs
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: RIDI.Ridibooks
+PackageVersion: 0.9.4
+PackageLocale: ko-KR
+Publisher: RIDI
+PackageName: Ridibooks
+License: Copyright © 2021 Ridibooks
+ShortDescription: Ridibooks PC Viewer
```
Comment (ItzLevvie) [inline manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml:10]:
```
ShortDescription: Ridibooks PC Viewer
```
Please write a better description.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml (hunk 2 above)
Comment (alexkoala) [reply manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml:10]: @ItzLevvie fixed

File: manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/r/RIDI/Ridibooks/0.9.4/RIDI.Ridibooks.locale.ko-KR.yaml]: This has to match what Control Panel sees.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see the latest review.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**